app/src/main/java/org/jellyfin/mobile/settings/SettingsFragment.kt

```diff
@@ -63,10 +65,23 @@ class SettingsFragment : Fragment() {
             titleRes = R.string.pref_video_player_type_title
             initialSelection = VideoPlayerType.WEB_PLAYER
             defaultOnSelectionChange { selection ->
+                externalPlayerChoicePreference.enabled = selection == VideoPlayerType.EXTERNAL_PLAYER
                 swipeGesturesPreference.enabled = selection == VideoPlayerType.EXO_PLAYER
                 backgroundAudioPreference.enabled = selection == VideoPlayerType.EXO_PLAYER
             }
         }
+        val externalPlayerOptions = listOf(
+            SelectionItem(ExternalPlayerType.MPV_PLAYER, R.string.external_player_mpv, R.string.external_player_mpv_description),
+            SelectionItem(ExternalPlayerType.MX_PLAYER_FREE, R.string.external_player_mx_player_free, R.string.external_player_mx_player_free_description),
+            SelectionItem(ExternalPlayerType.MX_PLAYER_PRO, R.string.external_player_mx_player_pro, R.string.external_player_mx_player_pro_description),
+            SelectionItem(ExternalPlayerType.VLC_PLAYER, R.string.external_player_vlc_player, R.string.external_player_vlc_player_description),
+            SelectionItem(ExternalPlayerType.SYSTEM_CHOICE, R.string.external_player_system_choice, R.string.external_player_system_choice_description),
+        ).filter { it.key.isEmpty() || isPackageInstalled(it.key) }
```

Comment:
Since we're building the list ourselves we can just exclude the empty key

Reply:
Also @Maxr1998 what happens if I select VLC, remove the app and then open the settings? The option wouldn't exist because `isPackageInstalled` returns false. Does the library crash or use the `initialSelection`?

Reply:
It shows nothing as selected then, actually.

Reply:
Not sure if that's the preferred behavior

Reply:
I'm not sure how to handle this case either, I made some changes but I don't know if it's the right way.